.appveyor.ps1

```diff
@@ -15,15 +15,11 @@ Write-Verbose "moving NetworkDirect headers.."
 move NetDirect\include\* include\windows
 Write-Verbose "done"
 
-$efaWinVersion="1.0.0"
-Write-Verbose "downloading efawin version ${efaWinVersion} files.."
-Invoke-WebRequest -Uri "https://github.com/aws/efawin/archive/refs/tags/v${efaWinVersion}.zip" -OutFile "efawin.zip"
-Write-Verbose "done"
-
-Write-Verbose "extracting efawin files.."
-$wd=$PWD.Path; & { Add-Type -A "System.IO.Compression.FileSystem"; [IO.Compression.ZipFile]::ExtractToDirectory("$wd\efawin.zip", "$wd"); }
+$efaWinVersion="v1.0.0"
+Write-Verbose "downloading efawin ${efaWinVersion} files.."
+git clone -b $efaWinVersion https://github.com/aws/efawin
```

Comment:
The `v1.0.0` branch ultimately needs to be deleted from aws/efawin, which would resolve this issue. I don't recommend merging this change if that happens.